samples/v1.0/Elements/FactSet.json

```diff
@@ -7,8 +7,8 @@
             "type": "FactSet",
             "facts": [
                 {
-                    "title": "Fact 1",
-                    "value": "Value 1"
+                    "title": "This is a long FactSet that should wrap the title",
```

Comment:
>"title": "This is a long FactSet that should wrap the title", [](start = 20, length = 61)

Should we make this a test file rather than changing our main FactSet example? I believe this is the one that will show up on [http://adaptivecards.io/explorer/FactSet.html](http://adaptivecards.io/explorer/FactSet.html) so we might want to keep it "nice looking"

Reply:
Good point. Moved it to test.